### PR TITLE
Added OneToMany mapping between Driver and RaceCar entities. Also ref…

### PR DESCRIPTION
**Pull request by raghav-codes**
Added OneToMany mapping between Driver and RaceCar entities. Also refactored the code - get end point now returning Entity list, instead of DTO list.

### CODE REVIEW

**Review by maanick — DISMISSED** [review]
Approved

File: src/test/java/com/example/speedwayapi/car/RaceCarServiceTest.java [hunk 1]
```diff
@@ -27,13 +27,13 @@
 
     @Test
     public void getAllRaceCarsTest(){
-        List<RaceCarDto> raceCarDtoList = new ArrayList<>();
+        List<RaceCarEntity> raceCarEntityList = new ArrayList<>();
```
Comment (davidgrieser) [inline src/test/java/com/example/speedwayapi/car/RaceCarServiceTest.java:30]: Nice refactor of the response of this endpoint.

**Review by davidgrieser — DISMISSED** [review]
LGTM

**Review by VaradharajanThiagarajan — APPROVED** [review]
LGTM

**Review by maanick — APPROVED** [review]
Approved

**Pull request merged**